kytos-init.sh

```diff
@@ -22,7 +28,13 @@ sed -i 's/WARNING/INFO/g' /etc/kytos/logging.ini
 test -z "$TESTS" && TESTS=tests/
 
 python3 scripts/wait_for_mongo.py 2>/dev/null
-python3 -m pytest $TESTS
+#python3 -m pytest --capture=tee-sys tests/test_e2e_31_of_lldp_loop_detection.py::TestE2EOfLLDPLoopDetection::test_001_loop_detection_disable_action
+#python3 -m pytest --capture=tee-sys tests/test_e2e_31_of_lldp_loop_detection.py::TestE2EOfLLDPLoopDetection::test_010_lldp_ignored_loops
+#python3 -m pytest --capture=tee-sys tests/test_e2e_31_of_lldp_loop_detection.py::TestE2EOfLLDPLoopDetection::test_020_reconfigure_ignored_loops
+python3 -m pytest --capture=tee-sys tests/test_e2e_31_of_lldp_loop_detection.py
```

Comment:
Looks like this part ended up accidentally committed could you revert this and only maintain the original line `python3 -m pytest $TESTS`?

Reply:
Yes, sure.

Reply:
@gretelliz, in this new commit a641ec7efc4ae939c00285d4e3d2232cabae43a5 it added again a line that is supposed to be reserved, looks like you were using it temporarily for testing, can you fix this?